tests/e2e/README.md

```diff
@@ -254,4 +254,8 @@ This section contains information about debugging osmosis's `e2e` tests.
         docker logs < container name/id >
     ```
 
-    Example: `docker logs osmo-test-a-node-prune-nothing-snapshot` will print logs emitted by container `osmo-test-a-node-prune-nothing-snapshot` to your standard output.
\ No newline at end of file
+    Example: `docker logs osmo-test-a-node-prune-nothing-snapshot` will print logs emitted by container `osmo-test-a-node-prune-nothing-snapshot` to your standard output.
+
+3. Viewing docker container logs when run osmosis's `e2e` tests in CI
+
+    When a failure occurs in CI, we have persists these log files of all container running as artifacts ####logs.tgz.
```

Comment:
```suggestion
    When a failure occurs in CI, we have persists these log files of all container running as artifacts file logs.tgz.
```

Reply:
It's outdated @p0mvn. I have a new commit after that